thirdeye/thirdeye-frontend/app/pods/components/composite-anomalies/entity-metrics-anomalies/template.hbs

```diff
@@ -0,0 +1,24 @@
+<div class="panel-group">
+  <div class="panel panel-default">
+    <div class="panel-heading">
+      <h4 class="panel-title">
+        {{title}}
+      </h4>
+    </div>
+    {{#if tableData.length}}
+      <div class="panel-body">
+        {{models-table
+          data=tableData
+          columns=tableColumns
+          showGlobalFilter=false
+          showColumnsDropdown=false
+          customClasses=customClasses
+        }}
+      </div>
+    {{else}}
+      <p class="composite-anomalies-no-records">
+        {{noRecords}}
```

Comment:
I see a few duplicate references to `noRecords`.

would it make sense to extract it to a small template {{composite-anomalies/no-records}}?

Reply:
I have moved everything common into a new component.